packages/dds/tree/src/feature-libraries/flex-tree/lazyField.ts

```diff
@@ -278,20 +278,8 @@ export class LazySequence extends LazyField implements FlexTreeSequenceField {
 	}
 }
 
-export class ReadonlyLazyValueField extends LazyField implements FlexTreeRequiredField {
```

Comment:
IIRC this used to be a base for both optional and value fields and provide read functionality. Optional no longer uses it, and the name is confusing since it sounds like it enforces readonlyness which it does not. It adds no value as has been removed.